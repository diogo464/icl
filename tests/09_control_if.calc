{
    let x = 10;
    if x == 10 { println "X is 10"; } else { println "X is not 10"; };
    let y = if x == 10 { 10 } else { 0 };
    if x == 11 {
        println "X is 11";
    } else if x == 12 {
        println "X is 12";
    } else {
        println "X is not 12 or 11";
    };
    println y;
};;