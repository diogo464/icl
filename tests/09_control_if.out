X is 10
X is not 12 or 11
10
